src/sentry/static/sentry/app/views/settings/components/tag.jsx

```diff
@@ -26,6 +28,7 @@ const TagTextStyled = styled(Box)`
   text-transform: lowercase;
   background-color: ${p =>
     p.priority ? p.theme.alert[p.priority].background : p.theme.offWhite2};
+  ${p => getBorder(p)};
```

Comment:
This can just be `${getBorder}`. right now you're just proxying the exact function signature :)